web404/get404.py

```diff
@@ -80,7 +80,7 @@ def get_page_status(page):
 
     virtual_browser = Grab()
     check = open_page(virtual_browser, url)
-    if check is not False and "200 OK" not in check.status:
+    if not check and not check.status and "200 OK" not in check.status:
```

Comment:
>>> a = None
>>> if not a and not a.status:
...     print "b"
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'status'